sandbox/public/iframed.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Alloy Test Page</title>
+    <style>
+      html,
+      body,
+      iframe {
+        height: 100%;
+      }
+      iframe {
+        width: 100%;
+      }
+    </style>
+  </head>
+  <body>
+    <iframe id="iframe" src="https://alloyio.com:3000" />
```

Comment:
Will this work once we host this file on `alloyio.com`? Are you trying to create a cross domain situation?

Reply:
This is meant to be used locally. Yes, I'm trying to create a cross domain iframe. https://github.com/adobe/alloy/wiki/Running-the-sandbox-locally-over-HTTPS-on-Mac-OS

Reply:
Maybe we could use a dummy GitHub page in the iFrame, that would work for both local and hosted on alloyio.com?

Reply:
I removed the port number. I'm still able to test by hosting alloyio.com locally on port 443. I added instructions to the wiki: https://github.com/adobe/alloy/wiki/Running-the-sandbox-locally-over-HTTPS-on-Mac-OS